src/store/store.js

```diff
@@ -1,8 +1,10 @@
 import { configureStore } from '@reduxjs/toolkit'
-import authReducer from 'features/authentication/authSlice'
+import authReducer from 'features/authentication/authSlice';
+import postReducer from 'features/posts/postSlice'
```

Comment:
Suggestion-You can try barrel exports and export from features/ Will be helpful as your slice code grows.

Reply:
ya but they are in different folders